docs/content/how-to/mce/Dual/hostedcluster/index.md

```diff
@@ -0,0 +1,5 @@
+A Hosted Cluster as the documentation [mentioned here](https://hypershift-docs.netlify.app/reference/concepts-and-personas/) it's just an OCP API endpoint managed by Hypershift but we will include also the term called HostedControlPlane in this concept to be more readable and easy to understand.
```

Comment:
```suggestion
A Hosted Cluster, as the documentation [mentions here](https://hypershift-docs.netlify.app/reference/concepts-and-personas/), is just an OCP API endpoint managed by Hypershift, but we will include also the term, _HostedControlPlane_, in this concept to be more readable and easy to understand.
```

Reply:
Humm I'm trying to not repeat things already has been documented in other section of the website. I want to put something like:

```
 Hosted Cluster, as mentioned in the documentation [here](https://hypershift-docs.netlify.app/reference/concepts-and-personas/), is essentially an OCP API endpoint managed by Hypershift. In this context, we will also include the term HostedControlPlane to enhance readability and comprehension. This terminology is further explained in the same [link](https://hypershift-docs.netlify.app/reference/concepts-and-personas/).
```